packages/terra-grid/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ Changelog
 
 Unreleased
 ----------
+### Changed
+* Removed variables file, breakpoints now imported from terra-breakpoints
```

Comment:
There are breaking changes we should callout here. I don't know if there is a standard for the changelog entry yet, but previously I had listed those changes under a `### Breaking Changes` heading

Reply:
AH, I wasn't aware of that. Will do!

Reply:
Here is what I link out for Changelog reference: https://keepachangelog.com/en/1.0.0/

Reply:
I think the Breaking Changes section adds more clarity to the change log and helps the changes stand out. For non-breaking changes I think the Changelog reference above is a good standard.

Reply:
There are a few other things we'll need to call out here. Including the removal of the `col` prop, the addition of the `enormous` prop,  and how the new breakpoint ranges affect this component. The old API has a different breakpoint mapping now

Reply:
@StephenEsser I agree we should add a `### Breaking Change` section with the list of changes and possibly link to the upgrade guide

Reply:
Sure thing, will add a list!